### PR TITLE
koordlet: improve peak predictor server

### PR DESCRIPTION
**Pull request by xigang**
### Ⅰ. Describe what this PR does

 Improve peak predictor server. 

### Ⅱ. Does this pull request fix one issue?
```
NONE
```

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/prediction/peak_predictor.go [hunk 1]
```diff
@@ -142,10 +151,10 @@ func (p *podReclaimablePredictor) AddPod(pod *v1.Pod) error {
 	}
 
 	if p.pods[string(pod.UID)] {
-		return fmt.Errorf("pod %v already exists", pod.UID)
-	} else {
-		p.pods[string(pod.UID)] = true
+		klog.V(6).Infof("Pod %s already exist in the pod reclaimable predictor", util.GetPodKey(pod))
+		return nil
```
Comment (hormes) [inline pkg/koordlet/prediction/peak_predictor.go:155]: It is not recommended to eat this error.

Comment (xigang) [reply pkg/koordlet/prediction/peak_predictor.go:155]: @hormes If this error message is not removed, will the amount of error messages be too much?

Comment (xigang) [reply pkg/koordlet/prediction/peak_predictor.go]: @hormes return error code rolled back.

**Review by saintube — APPROVED** [review]
/lgtm

**Pull request merged**